example/udf/src/main/java/org/apache/iotdb/udf/UDAFMKIdentify.java

```diff
@@ -0,0 +1,157 @@
+package org.apache.iotdb.udf;
+
+import org.apache.iotdb.udf.api.State;
+import org.apache.iotdb.udf.api.customizer.analysis.AggregateFunctionAnalysis;
+import org.apache.iotdb.udf.api.customizer.parameter.FunctionArguments;
+import org.apache.iotdb.udf.api.exception.UDFArgumentNotValidException;
+import org.apache.iotdb.udf.api.relational.AggregateFunction;
+import org.apache.iotdb.udf.api.relational.access.Record;
+import org.apache.iotdb.udf.api.type.Type;
+import org.apache.iotdb.udf.api.utils.ResultValue;
+import org.apache.iotdb.udf.table.MatchingKeyState;
+
+import org.apache.tsfile.utils.Binary;
+
+import java.nio.charset.Charset;
+import java.util.List;
+import java.util.Set;
+
+public class UDAFMKIdentify implements AggregateFunction {
+  int label;
+  int length;
+  double min_confidence;
+  double min_support;
+  String a;
+  int l = 0;
+
+  private FunctionArguments arguments;
```

Comment:
don't store this in your function, `UDAFMKIdentify.analyze` will only be called in query plan stage, and then other methods like `addInput` will be called in query execution stage. It's not the same object in these two stages, so the field you init in analyze won't be accessible to `addInput`